client/resources/kb/_technical/toFormat/otherDocumentFormat.dcl

```diff
@@ -2,13 +2,13 @@ function myFunction2(World world, Person me, Person her)
 --> local Person you
 --> domains Person
 --> action {
-    you     =new(Person, firstName, "Peter", lastName, "Pan")
+    you     =new(Person, firstName, "Peter", lastName, "Pan");
     me.friend:=     you;
-    world.execute(me)
+    world.execute(me);
     if(world.find(you)==true    ||world.find(her) !=false){
-        logInfo("Found somebody.")
+        logInfo("Found somebody.");
     }else{
-        logWarning("Find nobody")
+        logWarning("Find nobody");
```

Comment:
xD You’re the first victim of the breaking change.

Reply:
Yeah, it was quite painful.
For the tests it is mandatory to have everything updated, while in a project it's still OK and you can stick to an older version if you prefer.